controllers/validationController.js

```diff
@@ -41,14 +41,33 @@ exports.validateAdminPassword = (req, res, next) => {
     // console.log("adminPassword", adminPassword);
     const password = req.query.password
     // console.log("password", password);
-    if (password === adminPassword) {
-        // then good to go to players  
-        next();
-    } else {
+    
+    // get headers authorization
+    const authheader = req.headers.authorization;
+    console.log('req.headers.authorization', authheader);
+    // and distract only token password
+    const tokenPass = authheader.split(' ')[1]
+    console.log("tokenPass", tokenPass);
+
+    // get token password from frontend
+    if (password === adminPassword || tokenPass) {
```

Comment:
Shouldn't it be like the following?
```
if (password === adminPassword || tokenPass === adminPassword)
```
The current code only checks if the token exists or not.